web/src/main/java/org/bookkeepervisualmanager/api/resources/LedgersResource.java

```diff
@@ -101,4 +101,30 @@ private Integer convertParam(String s) {
         }
     }
 
+    /**
+     * Split a string into a Long list. If any value is invalid, return empty list
+     *
+     * @param s
+     * @return List
+     */
+    private List<Long> convertParamLongList(String s) {
+        if (s == null || s.trim().isEmpty()) {
+            return null;
```

Comment:
it is a code bad practice to return sometimes "null" and sometimes not.
Please return an empty list